gradlew.bat

```diff
@@ -25,7 +25,8 @@
 if "%OS%"=="Windows_NT" setlocal
 
 set DIRNAME=%~dp0
-if "%DIRNAME%" == "" set DIRNAME=.
+if "%DIRNAME%"=="" set DIRNAME=.
+@rem This is normally unused

```

Comment:
The file is auto-generated with ./gradlew wrapper after updating the config file to 7.6.1 so that the wrapper would actually run. I'd prefer not to touch it.